packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1087,13 +1092,20 @@ class ThemeData with Diagnosticable {
   /// start using new colors, typography and other features of Material 3.
   /// If false, they will use the Material 2 look and feel.
   ///
+  /// If true, the default Typography will be [Typography.material2021],
+  /// otherwise it will default to [Typography.material2014].
+  /// 
```

Comment:
This is a repeat of the section below. I reworded it to specifically call out that `useMaterial3` only effects the default typography if it is used in the constructor and not just set to true with a copyWith. The same will apply to the `splashFactory`. It will only take effect if `useMaterial3` is set in the constructor.

Reply:
Got it, thanks